packages/devextreme/testing/tests/DevExpress.ui/defaultOptions.tests.js

```diff
@@ -1332,6 +1333,13 @@ testComponentDefaults(ColorBox,
     }
 );
 
+testComponentDefaults(Chat,
+    {},
+    {
+        onMessageSend: null,
```

Comment:
What about default value of items option?

Reply:
this option is not processed here